odoo/openupgrade/doc/source/modules110-120.rst

```diff
@@ -217,9 +217,9 @@ missing in the new release are marked with |del|.
 +--------------------------------------------+-------------------------------------------------+
 |l10n_be_hr_payroll_fleet                    |                                                 |
 +--------------------------------------------+-------------------------------------------------+
-| |del| l10n_be_intrastat                    |                                                 |
+| |del| l10n_be_intrastat                    | Done. Merged into account                       |
```

Comment:
This text is still valid?

Reply:
No, sorry. Thanks. Maybe I would have seen this when squashing.